src/test/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverterTest.java

```diff
@@ -245,9 +255,11 @@ public void convertContentPackageDropContentTypePackagePolicy() throws Exception
             verifyFeatureFile(outputDirectory,
                             "asd.retail.all.json",
                             "asd.sample:asd.retail.all:slingosgifeature:0.0.1",
-                            Collections.singletonList("org.apache.felix:org.apache.felix.framework:6.0.1"),
-                            Collections.singletonList("org.apache.sling.commons.log.LogManager.factory.config~asd-retail"),
-                            Arrays.asList("asd.sample:asd.retail.apps:zip:cp2fm-converted:0.0.1", "asd:Asd.Retail.config:zip:cp2fm-converted:0.0.1"));
+                            Arrays.asList("org.apache.felix:org.apache.felix.framework:6.0.1"),
```

Comment:
changing from Collections.singletonList to Arrays.asList seems to be an unrelated change. please revert if it is not required for the test to pass